### PR TITLE
feat: Add doc link on each integration + style consistency

### PR DESCRIPTION
**Pull request by jbrisavoine**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [ ] If your changes affects code, did your write the tests?
- [x] Are tests passing? (`npm test` on both front/server)
- [x] Is the linter passing? (`npm run eslint` on both front/server)
- [x] Did you run prettier? (`npm run prettier` on both front/server)
- [x] If you are adding a new features/services, did you run integration comparator? (`npm run compare-translations` on front)
- [ ] Did you test this pull request in real life? With real devices? If this development is a big feature or a new service, we recommend that you provide a Docker image to the community ([french forum](https://community.gladysassistant.com/)/[english forum](https://en-community.gladysassistant.com/)) for testing before merging.
- [ ] If your changes modify the API (REST or Node.js), did you modify the API documentation? (Documentation is based on comments in code)
- [ ] If you are adding a new features/services which needs explanation, did you modify the user documentation? See [the GitHub repo](https://github.com/GladysAssistant/v4-website) and the [website](https://gladysassistant.com).
- [ ] Did you add fake requests data for the demo mode (`front/src/config/demo.js`) so that the demo website is working without a backend? (if needed) See [https://demo.gladysassistant.com](https://demo.gladysassistant.com).

NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open.

### Description of change

Following the request [https://community.gladysassistant.com/t/rendre-les-menus-homogene/8581/1](url):

- Add documentation link on each integration pages.
- Change icons and style for consistency
- Update `src/components/documentation/DeviceConfigurationLink.jsx` to be able to chose the path of the documentation

Before:
2023 12 07_PR_BEFORE

After:
2023 12 07_PR_AFTER

### CODE REVIEW

File: front/cypress/cypress.config.js [hunk 1]
```diff
@@ -65,5 +65,12 @@ module.exports = defineConfig({
         }
       }
     }
+  },
+
+  component: {
```
Comment (Pierre-Gilles) [inline front/cypress/cypress.config.js]: I'm not sure this is related to the current PR, if you want to modify the Cypress config, can you do it in a separate PR ? :)

File: front/cypress/support/component-index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
```
Comment (Pierre-Gilles) [inline front/cypress/support/component-index.html]: same

File: front/cypress/support/component.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands';
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```
Comment (Pierre-Gilles) [inline front/cypress/support/component.js]: same

File: front/package.json [hunk 4]
```diff
@@ -51,6 +51,8 @@
     "apexcharts": "^3.41.1",
     "axios": "^0.21.1",
     "classnames": "^2.3.1",
+    "common-js": "^0.3.8",
+    "commonjs": "^0.0.1",
```
Comment (Pierre-Gilles) [inline front/package.json]: Doesn't seem related to the current PR

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]
Great PR ! I have a few feedbacks on the included files in this PR, but nothing really serious :) 

For the Camera integration, it seems a bit too compact since the change, could we increase the width of each camera box ?

Screenshot 2023-12-08 at 09 57 25

Maybe we can put 2 cameras per row now?

File: front/cypress/cypress.config.js (hunk 1 above)
Comment (jbrisavoine) [reply front/cypress/cypress.config.js]: Sorry, it's my mistake. No need for the PR

File: front/cypress/support/component-index.html (hunk 2 above)
Comment (jbrisavoine) [reply front/cypress/support/component-index.html]: Same

File: front/cypress/support/component.js (hunk 3 above)
Comment (jbrisavoine) [reply front/cypress/support/component.js]: Same

File: front/package.json (hunk 4 above)
Comment (jbrisavoine) [reply front/package.json]: same same...

**Comment by jbrisavoine** [comment]
Fix size of camera box
2023 12 08_PR_FIX

File: front/src/routes/integration/all/nextcloud-talk/NextcloudTalk.jsx [hunk 5]
```diff
@@ -1,15 +1,42 @@
...
+            <div class="col-lg-3">
+              <h3 class="page-title mb-5">
+                <Text id="integration.nextcloudTalk.title" />
+              </h3>
+              <div>
+                <div class="list-group list-group-transparent mb-0">
+                  <DeviceConfigurationLink
+                    user={props.user}
+                    configurationKey="integrations"
+                    documentKey="nextcloudTalk"
```
Comment (Pierre-Gilles) [inline front/src/routes/integration/all/nextcloud-talk/NextcloudTalk.jsx:21]: This doesn't link to any documentation ! 

Maybe, let's put "nextcloud-talk" as the key and add a new documentation on the website?

File: front/src/routes/integration/all/openai/Layout.jsx [hunk 6]
```diff
@@ -1,10 +1,34 @@
...
+            <div class="col-lg-3">
+              <h3 class="page-title mb-5">
+                <Text id="integration.openai.title" />
+              </h3>
+              <div>
+                <div class="list-group list-group-transparent mb-0">
+                  <DeviceConfigurationLink
+                    user={user}
+                    configurationKey="integrations"
+                    documentKey="openai"
```
Comment (Pierre-Gilles) [inline front/src/routes/integration/all/openai/Layout.jsx:19]: This doesn't link to any documentation!

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]
Nice PR! :)

File: front/src/routes/integration/all/openai/Layout.jsx (hunk 6 above)
Comment (Pierre-Gilles) [reply front/src/routes/integration/all/openai/Layout.jsx:19]: Fixed

File: front/src/routes/integration/all/nextcloud-talk/NextcloudTalk.jsx (hunk 5 above)
Comment (Pierre-Gilles) [reply front/src/routes/integration/all/nextcloud-talk/NextcloudTalk.jsx:21]: Both documentation are live!

**Comment by Pierre-Gilles** [comment]
@all-contributors please add jbrisavoine for code

**Pull request merged**